recipes/qt/5.x.x/conanfile.py

```diff
@@ -109,7 +109,7 @@ class QtConan(ConanFile):
         "with_gstreamer": False,
         "with_pulseaudio": False,
         "with_dbus": False,
-        "with_gssapi": False,
+        "with_gssapi": True,
```

Comment:
```suggestion
        "with_gssapi": False,
```

I'd say given that this was false (and broken) for so long without anybody requesting it - it's likely not used by the majority of users, so probably best to leave the default and not complicate the dependency graph further.